BDArmory/Utils/FARUtils.cs

```diff
@@ -129,6 +129,27 @@ public static float GetFARcurrWingMass(Part part)
             }
             return -1;
         }
+        public static float GetFARWingSweep(Part part)
+        {
+            if (!hasFARWing) return 1;
```

Comment:
Doesn't having a sweep > 0 imply that the wing is not rectangular? So wouldn't this be better defaulting to 0 for non-swept wings?